tests/openwisp2/celery.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+from celery import Celery
+from celery.schedules import crontab
+
+# Replace 'openwisp2.settings' with path to your settings.py should be relative
+# from the location where celery command is executed.
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'openwisp2.settings')
+
+app = Celery('openwisp2')
+app.config_from_object('django.conf:settings', namespace='CELERY')
+app.autodiscover_tasks()
+
+
+@app.on_after_configure.connect
+def setup_periodic_tasks(sender, **kwargs):
+    from openwisp_radius.tasks import (
+        cleanup_stale_radacct,
+        deactivate_expired_users,
+        delete_old_postauth,
+        delete_old_radacct,
+        delete_old_users,
+    )
+
+    # You can configure parameters here to change when the commands will be
+    # automatically executed just like crontab.
+    sender.add_periodic_task(crontab(hour=0, minute=0), deactivate_expired_users)
+    sender.add_periodic_task(crontab(hour=0, minute=10), delete_old_users.s(365))
+    sender.add_periodic_task(crontab(hour=0, minute=20), cleanup_stale_radacct.s(365))
+    sender.add_periodic_task(crontab(hour=0, minute=30), delete_old_postauth.s(365))
+    sender.add_periodic_task(crontab(hour=0, minute=40), delete_old_radacct.s(365))
```

Comment:
can we keep consistency with [openwisp-monitoring and use CELERY_BEAT_SCHEDULE](https://github.com/openwisp/openwisp-monitoring/blob/master/tests/openwisp2/settings.py#L165-L172) please?